sbi/utils/user_input_checks_utils.py

```diff
@@ -25,17 +24,28 @@ def __init__(
         return_type: Optional[torch.dtype] = float32,
         batch_shape=torch.Size(),
         event_shape=torch.Size(),
-        validate_args=None,
+        validate_args=False,
+        lower_bound: Optional[Tensor] = None,
+        upper_bound: Optional[Tensor] = None,
     ):
         super().__init__(
             batch_shape=batch_shape,
             event_shape=event_shape,
             validate_args=validate_args,
         )
 
+        if not validate_args:
+            warnings.warn(
+                """Validate args is set to false. If there are constraints on
+                prior parameters (e.g., >0), consider passing arg constraints as
```

Comment:
I find this to be written confusingly. I suggest something like `If the prior has bounded support`

Reply:
Also, `consider passing arg constraints` is confusing. Why not say ``consider passing lower_bound and upper_bound`?